pkc

```diff
@@ -296,16 +296,43 @@ run_setup() {
     echo "Done!"
 }
 
-restore_all() {
-    archive_file=$1
-    # Extract tarball and restore images
-    docker exec -it micro-pkc-mediawiki /MediaWiki_Backup/extract_and_restore_images.sh -a "/backup/$archive_file" -w /var/www/html
-    # Restore database
-    docker exec -it micro-pkc-database /MediaWiki_Backup/restore_database.sh /backup/database.sql.gz
-    # Clean up database sql file
-    docker exec -it micro-pkc-database rm /backup/database.sql.gz
-    # Toggle read-only off
-    docker exec -it micro-pkc-mediawiki /MediaWiki_Backup/toggle_read_only_off.sh /var/www/html
+create_new_backup() {
+    sqlfile=$(docker run --rm --volumes-from micro-pkc-mediawikibackup busybox \
+        find /backup/latest -name '*.sql.gz' | head -1)
```

Comment:
We tested find output and the order was not predictable. We checked for alternatives and found 'sort' and 'ls' as a reliable alternative. @rht ht used ls -t (sort by time, newest first) as the previous solution.

Reply:
Nice catch and great advice. Will do.

Reply:
updated yesterday during pairing.